.github/workflows/py-test.yml

```diff
@@ -20,5 +20,5 @@ jobs:
     - name: Test with pytest
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        pip install -r test-requirements.txt
```

Comment:
please add as export-ignore to .gitattributes. We use that to create distribution zips and submit the PR to kodi repo